### PR TITLE
[Rebase & FF] Revert #1106 Clean up history to match release/202405

### PR DESCRIPTION
**Pull request by VivianNK**
## Description

PR #1106 pulled in other commits from release/202405. This reverts the PR's merge commit and cherry-picks the original commits.

- [ ] Impacts functionality?
- [ ] Impacts security?
- [ ] Breaking change?
- [ ] Includes tests?
- [ ] Includes documentation?

## How This Was Tested

Tested in release/2311 and dev/202405

## Integration Instructions

N/A

### CODE REVIEW

**Review by os-d — APPROVED** [review]

File: MdeModulePkg/Bus/Pci/PciBusDxe/PciLib.c
Comment (apop5) [inline MdeModulePkg/Bus/Pci/PciBusDxe/PciLib.c:1]: This change should not be in here. Can you make sure its removed?

File: NetworkPkg/UefiPxeBcDxe/PxeBcBoot.c
Comment (apop5) [inline NetworkPkg/UefiPxeBcDxe/PxeBcBoot.c:1]: Same with this one, this was cherry-picked from a later point in edk2

File: NetworkPkg/UefiPxeBcDxe/PxeBcImpl.c
Comment (apop5) [inline NetworkPkg/UefiPxeBcDxe/PxeBcImpl.c:1]: Same with this one, this was cherry-picked from a later point in edk2

**Review by apop5 — CHANGES_REQUESTED** [review]

File: MdeModulePkg/Bus/Pci/PciBusDxe/PciLib.c
Comment (VivianNK) [reply MdeModulePkg/Bus/Pci/PciBusDxe/PciLib.c:1]: https://github.com/microsoft/mu_basecore/commit/456eada7f1601d65326cd3030d8b66c631980c68#diff-a01943b670a19f44f40fd9f4c640c1d9ff4208d542034ebe432f09f35e7f0cfa

This is reverting the commit that brought this change in erroneously

**Review by apop5 — APPROVED** [review]

**Pull request closed without merging**